src/ophys_etl/workflows/app_config/app_config.py

```diff
@@ -199,6 +203,7 @@ class _PipelineSteps(ImmutableBaseModel):
     motion_correction: _MotionCorrection = Field(default=_MotionCorrection())
     segmentation: _Segmentation = Field(default=_Segmentation())
     trace_extraction: _TraceExtraction = Field(default=_TraceExtraction())
+    demix_traces: _DemixTraces = Field(default=_DemixTraces)
```

Comment:
this will break. Please look at the pattern in the other attributes. It sets the default to the instantiated object.

There was an issue with a numpy upgrade which prevented tests from running. It's been fixed in workflow_v2. Please merge that branch into this one so tests can run.

Reply:
Cool, was planning on rebasing before merging into workflow. Also I see the issue above. Will fix and ping again for a review.